controllers/fenceagentsremediation_controller_test.go

```diff
@@ -133,29 +133,37 @@ var _ = Describe("FAR Controller", func() {
 			It("should have finalizer and taint", func() {
 				farNamespacedName := client.ObjectKey{Name: node01, Namespace: defaultNamespace}
 				nodeNamespacedName := client.ObjectKey{Name: node01}
```

Comment:
This is a bit confusing, since namespace isn't relevant  in the context of nodes

Reply:
I usually use something like `nodeKey`: it represents the type (ObjectKey) and it doesn't matter if it has a namespace or not 🙂